template/.devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "forwardPorts": [
```

Comment:
Does this key have to be in the student's `.devcontainer.json`, in addition to the one we're using to prebuild the image? (If so, should the latter be removed then?)